userguide/tutorials/getting_started.adoc

```diff
@@ -71,27 +109,29 @@ services:
       - MYSQL_ROOT_PASSWORD=killbill
 ----
 
-and run:
+Now place this file in your current directory and run:
 
 [source,bash]
 ----
 docker-compose up
 ----
 
-3 containers will start:
+If all goes well 3 containers will start:
```

Comment:
If you have any tips, based on what you have seen, on potential issues and how to track them, you could add some a `troubleshooting` section.

Reply:
I'm not sure if I know enough to do this yet, but I will work on it